src/components/FilterList/hooks.ts

```diff
@@ -9,3 +9,33 @@ export function useFilterListProvider() {
   }
   return context;
 }
+
+/**
+ * Generates a function that calculates the available height for the `FilterList` component
+ *
+ * @param minHeight The minimum height the `FilterList` can be
+ * @param bottomMargin The amount of margin to the bottom of the panel, this value is
```

Comment:
```suggestion
 * @param bottomMargin The margin to the bottom of the panel. This value is
```
But blease also clarify what "The margin to the bottom of the panel" means. The margin from what?? :)